go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/gin-gonic/gin
 
-go 1.18
+go 1.20
```

Comment:
please revert it, thanks!

Reply:
if the version in go.mod is not 1.20, the unit tests will fail.